frontend/src/components/Login.js

```diff
@@ -61,7 +61,7 @@ class Login extends Component {
 
   render() {
     const errors = (this.state.error !== '') ?
```

Comment:
I know this is not from the the current branch, but let's have this condition just as `const errors = this.state error ?`.. I think it works and easier to read